rmw_dds_common/include/rmw_dds_common/context.hpp

```diff
@@ -27,16 +27,27 @@
 namespace rmw_dds_common
 {
 
+/// Structure of data that a Context will need in any rmw implementation
```

Comment:
@ahcorde @ivanpauno nit:
```suggestion
/// Base data structure that a Context will need in any DDS-based RMW implementation
```

Reply:
I'd also mention that this implies one participant per context.

Reply:
> I'd also mention that this implies one participant per context.

That was already mentioned in the next line.